packages/fuels-core/src/code_gen/custom_types/utils.rs

```diff
@@ -145,22 +145,27 @@ pub fn extract_custom_type_name_from_abi_property(prop: &TypeDeclaration) -> Res
 pub(crate) fn param_type_calls(field_entries: &[Component]) -> Vec<TokenStream> {
     field_entries
         .iter()
-        .map(|Component { field_type, .. }| {
-            let type_name = &field_type.type_name;
-            let parameters = field_type
-                .generic_params
-                .iter()
-                .map(TokenStream::from)
-                .collect::<Vec<_>>();
-            if parameters.is_empty() {
-                quote! { <#type_name>::param_type() }
-            } else {
-                quote! { #type_name::<#(#parameters),*>::param_type() }
-            }
-        })
+        .map(|Component { field_type, .. }| single_param_type_call(field_type))
```

Comment:
I would have called it `param_type_call` singular. 

But then again those plural 's'-es can kind of hide in plain sight, so this is probably more in your face can't mistake it for the other one.

Reply:
Yup, that was my thought process too